mopidy_spotify/playlists.py

```diff
@@ -56,9 +56,15 @@ def _get_playlist(self, uri, as_items=False, with_owner=False):
         )
 
     @staticmethod
-    def _span(p, xs):  # like haskell's Data.List.span
-        i = next((i for i, v in enumerate(xs) if not p(v)), len(xs))
-        return xs[:i], xs[i:]
+    def _split_ended_movs(value, movs):
+        def _span(p, xs):
+            # Returns a tuple where first element is the longest prefix
+            # (possibly empty) of list xs of elements that satisfy predicate p
+            # and second element is the remainder of the list.
+            i = next((i for i, v in enumerate(xs) if not p(v)), len(xs))
+            return xs[:i], xs[i:]
+
+        return _span(lambda e: e[0] < value, movs)
```

Comment:
I don't see the point in having to think about a lamdba in order to understand this function, but OK, at least it should be easier to test now.